src/nr_openai_observability/build_events.py

```diff
@@ -76,7 +83,7 @@ def build_events(response, request, response_headers):
     return {"messages": messages, "completion": completion}
 
 
-def build_error_events(request, error):
+def build_completion_error_events(request, error):
     completion_id = str(uuid.uuid4())
```

Comment:
We should put it under a function and use it across the code

Reply:
will do